pkg/provider/aws/resources/ecs.go

```diff
@@ -261,6 +267,14 @@ func (s *EcsService) Id() core.ResourceId {
 	}
 }
 
+func (td *EcsService) DeleteCriteria() core.DeleteCriteria {
+	return core.DeleteCriteria{
+		RequiresNoUpstream:     true,
+		RequiresNoDownstream:   true,
+		RequiresExplicitDelete: true,
+	}
```

Comment:
`RequiresNoUpstreamOrDownstream`? There are a few others like this in other files.

Reply:
it would just default to false which is whats intended here because the combination of requires no upstream and requires no downstream will cover both scenarios